lib/graphql/persisted_queries/error_handlers.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require "graphql/persisted_queries/error_handlers/base_error_handler"
+require "graphql/persisted_queries/error_handlers/default_error_handler"
+
+module GraphQL
+  module PersistedQueries
+    # Contains factory methods for error handlers
+    module ErrorHandlers
+      def self.build(handler, options = nil)
+        if handler.is_a?(ErrorHandlers::BaseErrorHandler)
```

Comment:
we probably could allow a callable object (like lambda) here

Reply:
Yep, good idea!  Added.